Packages/com.unity.inputsystem/InputSystem/Editor/UITKAssetEditor/InputActionsEditorWindow.cs

```diff
@@ -49,9 +52,26 @@ public static bool OpenAsset(int instanceId, int line)
             //       without forcing a checkout.
             var obj = EditorUtility.InstanceIDToObject(instanceId);
             var asset = obj as InputActionAsset;
-            if (asset == null)
-                return false;
 
+            string actionMapToSelect = null;
+            string actionToSelect = null;
+
+            // Means we're dealing with an InputActionReference, e.g. when expanding the an .input action asset
+            // on the Asset window and selecting an Action.
+            if (asset == null)
+            {
+                var actionReference = obj as InputActionReference;
+                if (actionReference != null && actionReference.asset != null)
+                {
+                    asset = actionReference.asset;
+                    actionMapToSelect = actionReference.action.actionMap?.name;
```

Comment:
Should this be checking if action is null too 
actionReference.action?.actionMap?.name;